internal/runbits/requirements/requirements.go

```diff
@@ -174,6 +174,10 @@ func (r *RequirementOperation) ExecuteRequirementOperation(requirementName, requ
 			}
 			return locale.WrapInputError(err, "package_ingredient_alternatives", "", requirementName, strings.Join(suggestions, "\n"))
 		}
+		if name := packages[0].Ingredient.Name; name != nil && requirementName != *name {
```

Comment:
This makes sense, you could add an extra `strings.EqualFold` check to ensure that `requirementName` and `name` are actually the same in a case insensitive way... though `SearchIngredientsStrict` _should_ cover that. This would just be another check to ensure they actually match.